Packs/Base/Scripts/DBotShowClusteringModelInfo/DBotShowClusteringModelInfo.py

```diff
@@ -0,0 +1,75 @@
+from CommonServerPython import *
+
+# CONSTS
+BASE_INCIDENT_COLUMNS = ['id', 'created', 'name', 'Group name', 'Group id']
+
+
+def show_model_info(model_name):
+    success = False
+    start = datetime.now()
+    model = None
+
+    while not success and datetime.now() - start < timedelta(minutes=10):
+        res = demisto.executeCommand("getMLModel", {"modelName": model_name})
+        if is_error(res):
```

Comment:
10 min to load the model seems very long no ?

Reply:
it's because we are waiting to model fiinish traning if it's first time the widget is running

Reply:
ok